packages/opentelemetry-propagator-jaeger/src/JaegerPropagator.ts

```diff
@@ -47,12 +47,16 @@ const UBER_BAGGAGE_HEADER_REGEX = /^uberctx-(.+)/i;
  */
 export class JaegerPropagator implements TextMapPropagator {
   private readonly _jaegerTraceHeader: string;
-
-  /**
-   * @param {string} [customTraceHeader="uber-trace-id"] - HTTP header to inject\extract trace from.
-   **/
-  constructor(customTraceHeader?: string) {
-    this._jaegerTraceHeader = customTraceHeader || UBER_TRACE_ID_HEADER;
+  private readonly _jaegerBaggageHeaderPrefix: string;
+
+  constructor(config?: JaegerPropagatorBackwardCompatibleConfig) {
```

Comment:
We don't have to expose the backwards compatibility hackyness to our users if we use overloads

```suggestion
  constructor(customTraceHeader?: string);
  constructor(config?: JaegerPropagatorConfig);
  constructor(config?: JaegerPropagatorConfig | string) {
```

Reply:
Fixed. Thank you.